.github/workflows/ci.yml

```diff
@@ -43,3 +43,4 @@ jobs:
       - uses: prometheus/promci@v0.0.2
       - uses: ./.github/promci/actions/setup_environment
       - run: make
+      - run: git diff --exit-code
```

Comment:
This step will fail if the artifacts are not updated. The resulting behavior is the same as the legacy UI.

Tested as part of this PR, the test is visible in this PR's commit history.